### DISCUSSION TITLE
用於軍事的穿戴裝置-MDO

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#33] [General]
**內容大綱:**
從 1861 年美國南北戰爭開始，士兵就使用金屬製兵籍牌，俗稱狗牌（Dog Tag），上面記著士兵姓名及血型等重要資訊，以便急救或收拾遺體時辨認。
除了個人**資訊辨識**，美軍希望即時掌握前線士兵**生理資訊和周遭環境**，因此**美軍近期與NextFlex 公司合作**，透過彈性電子零件技術，開發稱為**多領域行動穿戴裝置**（Multi-Domain Operations wearable，**MDO**）。

造型輕薄可直接縫在官兵制服內，隨時連線指揮中心，內建微型**無線電發送裝置**及各種感測器可即時偵測每位士兵體力、生理傷病、周遭環境、天氣、濕度等氣象資訊，MDO可做到戰場即時情報分析，如裝**聲源感測器**，若發生槍響，由指揮中心 AI 系統透過三角定位找出敵方位置，並透過槍響聲紋辨識，分析發射的武器類型。而不只軍事方面，醫療和救災領域也有極大幫助。
雖說 MDO 有許多先進功能，但若 MDO 被打壞了，或穿戴者炸得四分五裂，最後還是得靠老方法才找得回來。

**心得:**
我覺得這個穿戴型裝置產生的利大於弊，而我也從未想過這樣的科技可以應用在軍事方面，彷彿像是電影才會出現的先進設備！而比起軍事用途，我認為更重要的是醫療，若是上場的士兵身理有狀況而自己不知情，不僅會造成整體軍隊的負擔，也讓自己先病倒在戰場上。雖然能對即時軍情有很大貢獻，但還是希望不要有戰爭發生比較好！

**新聞來源:**
https://technews.tw/2021/10/05/us-army-and-nextflex-are-developing-smart-dog-tags/
![image](https://user-images.githubusercontent.com/90883526/136219016-632f1763-602a-4ea0-96c9-70297d9145a4.png)

**Comment by jim11261314** [comment]
感覺這技術也可以用來放在社區的老年者
這樣在家中跌倒或心血管出了問題，可以即時救援

**Comment by LanningYang** [comment]
如用於救災 想必亦能減少傷亡
